src/modules/addEditRemove.js

```diff
@@ -0,0 +1,127 @@
+import Icon from '../assets/images/icon.png';
+import Trash from '../assets/images/trash.png';
+
+export const todoList = document.querySelector('#todoListPlaceholder');
+const addTaskArea = document.querySelector('#addTaskArea');
+export const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+
+function updateLocalStorage() {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+export function displayList(obj) {
+  const item = document.createElement('li');
+  item.className = 'ListItem';
+  item.id = obj.index;
+  item.innerHTML = `
+  <input type="checkbox" class="checkboxBtn" id="a${obj.index}">
+  <textarea name="itemDescription" class="itemTextArea" id="b${obj.index}" rows="1" cols="50">
+  ${obj.description}
+  </textarea>
+  <div class="imgDiv">
+  <img src="${Icon}" class="Dots" id="d${obj.index}">
+  </div>
+  `;
+  todoList.appendChild(item);
+  if (obj.completed === true) {
+    const checkbox = document.querySelector(`#a${obj.index}`);
+    checkbox.checked = true;
+  }
+}
+
+export function clearList() {
+  while (todoList.firstChild) {
+    todoList.removeChild(todoList.firstChild);
+  }
+}
+
+export const addTask = () => {
+  if (addTaskArea.value !== '') {
+    const newTask = {
+      description: addTaskArea.value,
+      completed: false,
+      index: tasks.length,
```

Comment:
Make sure the property index matches the new array length, as the project requirements specify. For example, the index of the 5th task in the list should be 5. Right now, the first index is 0.

What it currently is:
<img width="587" alt="image" src="https://github.com/IsmaelMastronardi/Todo_List/assets/52934128/5a464dc2-43e7-4661-a239-b366af10b38f">

What the project requires:
<img width="691" alt="image" src="https://github.com/IsmaelMastronardi/Todo_List/assets/52934128/abf29d32-4ecb-4fa2-b776-80a2dc5e7d47">